### ISSUE TITLE
takeover.

### ISSUE THREAD

**Issue by ghost** [dbt-labs/tap-liveperson#3]
Hi,
I'm interested in taking this over if it's the latest and greatest version.  Is this branch publishing to pypi?

Gord

**Comment by drewbanin** [comment]
Hi @gord-nuttall-adswerve - thanks for opening this issue! This repo is the latest version of the tap-liveperson codebase, at least as far as I am aware. If you'd like to mirror the codebase into your GitHub org, I can add a note to the README pointing folks to the new repo.

It does look like this package is published to [pypi](https://pypi.org/project/tap-liveperson/), though the last release was in July, 2020. I am not 100% sure what the right protocol for transferring ownership of a pypi package is, though my guess is that you might want to set up your own release process with a different package name on pypi.

Thanks so much!

**Comment by ghost** [comment]
I have forked the repo to here:
adswerve/tap-liveperson

We'd like to keep the same package name in pypi.  Do you know who this organization is: https://pypi.org/user/fishtown-analytics/.

Please, if possible, try to add PyPI account (https://pypi.org/user/Adswerve/) as a collaborator to PyPI project (https://pypi.org/project/tap-liveperson/)

**Comment by drewbanin** [comment]
hey @gord-nuttall-adswerve -- thanks for doing that! I'll update the readme to point to this fork now.

We are the owners of the `fishtown-analytics` pypi org, though I think our preference would be to _not_ transfer ownership to the Adswerve pypi org. Is it possible for your team to add a new package on pypi with a different name instead? I can make sure to update the README pointing to your package accordingly.

**Comment by ghost** [comment]
Why the hesitation?  Of course, it's possible to use a different name, but all taps within the singer.io framework use the tap-<datasource> format.  It's just not following the naming convention and making for a confusing user experience.

**Comment by drewbanin** [comment]
I agree - the user experience would be pretty confusing, though I think it's sort of unconventional to transfer ownership of a package to someone else without a lot of communications to active users. I'd be supportive of transferring this over at some point in the future, though I think it would be good to establish you fork as the new upstream before we change over pypi to point to your codebase. Does that sound alright to you?

**Comment by ghost** [comment]
That's fair.  We'll likely be updating the tap here shortly.  Do you have previous unit tests or a QA process you can run on our updated version to ensure nothing broke?  Perhaps that's a way to ensure the PyPI version/code remains stable during the handoff?

**Comment by drewbanin** [comment]
We (very unfortunately) do not have any unit tests or automated QA processes for this tap. In the past, we've done manual testing before releasing this package. This is one of the many reasons why we've deprecated this particular repo & why we're so excited that you're willing to take over as the upstream here!

I just updated the repo README point to your fork of the repo: https://github.com/dbt-labs/tap-liveperson/blob/master/README.md#this-repo-is-no-longer-actively-maintained
